README.md

```diff
@@ -1,2 +1,65 @@
-# tiny-email-sender
-A tiny node js server that can help to send email from SMTP protocol
+# Sendver
+
+[![License](https://img.shields.io/github/license/Los-Crackitos/Sendver)](LICENSE)
+
+Sendver is a tiny email sender API that allow users to send email throught http POST request. You can use an SMTP provider or mailgun API.
+
+## Installation
+
+To install Sendver, you'll need to run the docker image from docker hub (link). You must set environment variable, than can allow you to send email from SMTP server or by mailgun API.
+
+`docker run -p 8080:3000 --env MAILGUN_API_KEY=MYAPIKEY --env MAILGUN_DOMAIN=MYDOMAIN sendver `
+
+Here is the list of the environments variables :
+
+| Key             | Description                                                    |
+| --------------- | -------------------------------------------------------------- |
+| PORT            | By default, is `3000`.                                         |
+|                 |                                                                |
+| SMTP_URL        | Is the SMTP server url that can be used to send email.         |
+| SMTP_PORT       | Is the SMTP server port that can be used to send email.        |
+| SMTP_USERNAME   | Is the SMTP server username used to connect to smtp server.    |
+| SMTP_PASSWORD   | Is the SMTP server password used to connect with the username. |
```

Comment:
maybe remove the `is` in front of each sentences, it seems like weird question due to the starting `is`